client/services/Hooks.ts

```diff
@@ -99,11 +99,27 @@ function withAuthModal(isLoggedIn: boolean, cb: (...any) => any) {
   };
 }
 
+function pressEsc(handler: (e: KeyboardEvent) => void) {
+  useEffect(() => {
+    const handleEsc = (event) => {
+      if (event.code === 27) {
```

Comment:
Did you test your code after the changes? `event.code` is not a number, but a string. Pretty sure this code doesn't work.

![image](https://user-images.githubusercontent.com/11327761/148270165-ecf116f8-9194-436c-ab3a-583616cdaba3.png)